### PR TITLE
Added my profile

### PR DESCRIPTION
**Pull request by PritamSarbajna**
## Changes proposed

I made a new file called PritamSarbajna.json
there I added my profile

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Screenshots

Screenshot (194)

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/1598"></a>

### CODE REVIEW

File: public/data/PritamSarbajna.json [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+        "title": "Got 1st prize in a website building competition",
+        "date": "July 2022",
+        "icon": "book",
+        "color": "purple",
+        "description": "I made a NGO website with django and tailwind CSS and got 1st prize."
+      }
+      
+    ]
+  }
+
```
Comment (krshkodes) [inline public/data/PritamSarbajna.json:83]:
```suggestion

```

File: public/data/PritamSarbajna.json [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+        "description": "I enrolled into BTech course in IT department at AEC"
+      },
+      {
+        "title": "Got 1st prize in a website building competition",
+        "date": "July 2022",
+        "icon": "book",
+        "color": "purple",
+        "description": "I made a NGO website with django and tailwind CSS and got 1st prize."
+      }
+
```
Comment (krshkodes) [inline public/data/PritamSarbajna.json:80]:
```suggestion
```

**Review by krshkodes — CHANGES_REQUESTED** [review]
Only some formatting issues 🤔

File: public/data/PritamSarbajna.json [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+      },
+      {
+        "title": "Got 1st prize in a website building competition",
+        "date": "July 2022",
+        "icon": "book",
+        "color": "purple",
+        "description": "I made a NGO website with django and tailwind CSS and got 1st prize."
+      }
+    ]
+  }
```
Comment (krshkodes) [inline public/data/PritamSarbajna.json:81]:
```suggestion
  }

```

**Review by krshkodes — COMMENTED** [review]
Missing an endline

**Review by Panquesito7 — APPROVED** [review]
Looks good. Thanks for contributing! 🎉


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good&nbsp; | &nbsp;**Tags:** Maintainable

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to merge!

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good&nbsp; | &nbsp;**Tags:** Reusable

**Review by EmmaDawsonDev — APPROVED** [review]

**Pull request merged**